### ISSUE TITLE
How do I check status of emqx server when use docker ?

### ISSUE THREAD

**Issue by bigboxthailand** [emqx/emqx-docker#72]
### Docker version
18.06.1

### System
Raspbian

### Hardware
Raspberry pi 3b+

I am newbie , now I can start and stop and pub sub the message but I don't know how to check status  like ./bin/emqx_ctl status ?

**Comment by bigboxthailand** [comment]
@terry-xiaoyu 
Does it restart automatically ?

**Comment by terry-xiaoyu** [comment]
No, it doesn't. You should use the [restart policy](https://docs.docker.com/config/containers/start-containers-automatically/#use-a-restart-policy) of the docker container.
